README.md

```diff
@@ -53,28 +111,47 @@ You should see something like this:
 
 ![yarn workspace eth hardhat:prod deploy](img/hardhat_prod_deploy.png)
 
-> Dark Forest uses [yarn workspaces](https://yarnpkg.com/features/workspaces)
-
 To find out more information about the `contracts` submodule, you can look here:
 [https://github.com/darkforest-eth/eth](https://github.com/darkforest-eth/eth).
 
-### Step 2: deploy client website
+### Step 2: Deploy client website
 
 To deploy the website interface, you may either self-host, or use the same infra that we use -
 [Netlify](https://www.netlify.com/). This is a simple option, and free for up to some amount of
 gigabytes of bandwidth per month, which has often been enough for us.
 
-To deploy to Netlify, you can run the following command:
-
-```bash
-yarn workspace client deploy:prod
-```
-
-You should see something like this:
-
-![yarn workspace eth hardhat:prod deploy](img/netlify_prod_deploy.png)
-
-### Step 3: allow player addresses
+To use Netlify:
+
+- Make a new [Netlify account](https://app.netlify.com/signup) using your Github profile.
+- [Import](https://app.netlify.com/start) your Github client repo as a new Netlify site.
+- Install the Netlify CLI
+    - `npm install netlify-cli -g`
+- Login to your account
+    - `netlify login`
+- Connect to your app
+    - `netlify link`
+    
+    ![link](img/link.png)
+    
+- Choose *Enter a site ID*
+    
+    ![Screen Shot 2022-01-15 at 12.35.15 PM.png](img/site_id.png)
+    
+- Input the id that you find by clicking on your new site on Netlify. In this example, the id is `jolly-hermann-3947ca`
+- Now that your site is linked, you can deploy to Netlify from the CLI.
+    - `yarn workspace client deploy:prod`
+    - If successful, you will see something like this.
+        
+        ![deploy_success](img/netlify_deploy.png)
+        
+    - *Note: To do this, you will have to turn on automatic deploys for your site. This means that if you push an update to your main Git branch, Netlify will attempt to redeploy the site. This will fail, because the client relies on the most recent versions of the packages you have created, but Netlify will try to install the older versions from npm instead of locally.*
```

Comment:
Why don't you turn off the deploys? Or instead recommend the users to deploy a subdirectory of darkforest-local?